modules/android/pages/gs-prereqs.adoc

```diff
@@ -1,20 +1,27 @@
+
 = Preparing for Couchbase Lite on Android
 :page-aliases: start/java-android-gs-prereqs.adoc
-ifdef::show_edition[:page-edition: {release}]
-ifdef::prerelease[:page-status: {prerelease}]
 :page-role:
 :description: Prerequisites for the installation of Couchbase Lite
 :keywords: edge nosql api android java
 
-include::partial$_set_page_context_for_android.adoc[]
 
-:param-abstract: Laying out some of the pre-requisites and preparatory steps to be considered before installing Couchbase Lite for {param-name}
-// DO NOT EDIT
```

Comment:
You have no idea how much I've enjoyed deleting all the "DO NOT DELETE" warnings...